build/commands/lib/util.js

```diff
@@ -517,15 +517,20 @@ const util = {
     let msBuild = ''
     if (vsVersion === '2017') {
       msBuild = path.join(vsPath, 'MSBuild', '15.0', 'Bin', 'MSBuild.exe')
-    } else if (vsVersion === '2019') {
+    } else if (vsVersion === '2019' || vsVersion === '2022') {
```

Comment:
should this just be `} else {` because all versions > 2017 use Current?

Reply:
Fixed up as discussed via Slack.